pints/_mcmc/_global_adaptive_covariance.py

```diff
@@ -0,0 +1,62 @@
+#
+# Base class for Adaptive covariance MCMC methods
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2018, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+
+
+class GlobalAdaptiveCovarianceMCMC(pints.AdaptiveCovarianceMCMC):
+    """
+    Base class for single chain MCMC methods that globally adapt a
```

Comment:
A lot of these files seem to do line breaks arbitrarily instead of at 79 chars

Reply:
Have changed some of the text so that it runs to 79 instead.